core/unit_test/view/TestBasicView.hpp

```diff
@@ -0,0 +1,219 @@
+//@HEADER
+// ************************************************************************
+//
+//                        Kokkos v. 4.0
+//       Copyright (2022) National Technology & Engineering
+//               Solutions of Sandia, LLC (NTESS).
+//
+// Under the terms of Contract DE-NA0003525 with NTESS,
+// the U.S. Government retains certain rights in this software.
+//
+// Part of Kokkos, under the Apache License v2.0 with LLVM Exceptions.
+// See https://kokkos.org/LICENSE for license information.
+// SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+//
+//@HEADER
+
+#include <gtest/gtest.h>
+
+#include <Kokkos_Core.hpp>
+#include <type_traits>
+#include "experimental/__p0009_bits/dynamic_extent.hpp"
+#include "experimental/__p2642_bits/layout_padded_fwd.hpp"
+#ifndef KOKKOS_IMPL_PUBLIC_INCLUDE
```

Comment:
It really shouldn't be defined at this point, right?

Reply:
Yeah probably not.

Reply:
fixed